spec/models/friendship_spec.rb

```diff
@@ -2,8 +2,9 @@
 
 RSpec.describe Friendship, type: :model do
   it 'Create a new data container' do
-    friendship = Friendship.new(user_id: 1, friend_id: 2)
-    expect(friendship).to be_a_new(Friendship)
+    @friendship = Friendship.new(user_id: 1, friend_id: 2)
+    @friendship = Friendship.new(user_id: 2, friend_id: 1)
```

Comment:
#### [OPTIONAL]

Declaring an instance variable here is unnecessary as you are not using them anywhere else. Please convert them to normal variables.